.cirrus.yml

```diff
@@ -268,6 +307,23 @@ task:
   << : *FLUTTER_UPGRADE_TEMPLATE
   << : *MACOS_ARM_TEMPLATE
   matrix:
+    - name: macos-custom_package_tests
```

Comment:
The weird diff here and below is because I noticed that I had accidentally put this in the "iOS tasks" group when I added groups to this file, but we don't actually know what the target platform(s) is/are for custom test scripts because they can do anything.